admin/app/styles/components/complementary-certifications/attach-target-profile.scss

```
@@ -0,0 +1,25 @@
+.complementary-certification-attach-target-profile {
+  display: flex;
+  flex-direction: column;
+  gap: 15px;
+
+  &__header {
+    color: $pix-neutral-90;
+    font-size: 28px;
```

Est-ce sur les size on utilises aussi le design system quand c'est possible (comme pour les couleurs ?)

```suggestion
    font-size: %pix-title-s;
```

https://ui.pix.fr/?path=/docs/utiliser-pix-ui-design-tokens-typographie--docs